src/flamegraph/mod.rs

```diff
@@ -9,6 +9,7 @@ use std::io::prelude::*;
 
 mod color;
 mod merge;
+mod palettes;
```

Comment:
I think we'll want this as a submodule of `color`. I agree it's a little annoying to add a subdirectory for color, but I do think the module hierarchy makes more sense that way. We'd then have:
```
src/flamegraph/
  color/mod.rs <- what is now in color.rs
  color/palettes.rs <- what is now in palettes.rs
```

Reply:
Seems fine. We can add a `palette_map.rs` file in here if we move the future `PaletteMap` struct to its own file.

Reply:
[Moved](https://github.com/jonhoo/inferno/pull/32/commits/4a774c8463b75acace1f20018da4f3b1ba4dff0f)

Reply:
Ah, yes, moving `PaletteMap` too seems like a great idea!

Reply:
[Moved `palette_map` functions](https://github.com/jonhoo/inferno/pull/32/commits/1ea68876e45d4afefc6674dbf50c81ed4e96d764)